docs/sources/writing-guide/documentation-structure/index.md

```diff
@@ -12,21 +12,52 @@ keywords:
+Before you begin contributing to documentation, it is important to understand the structure of the content.
 
-Generally, the IA determines how content is:
+According to STC’s [Information Design and Information Architecture: Why Technical Communicators Should Care About These Fields](https://www.stc.org/intercom/2022/05/information-design-and-information-architecture-why-technical-communicators-should-care-about-these-fields/),
+information architecture is "...the practice of organizing, structuring, and labeling information to it’s easy to find, use, and understand..."
+
+Generally, documentation structure determines how content is:
 
 - Titled
 - Grouped
 - Combined (or not combined) with other, related content
 
 The following examples are based on the Grafana OSS and Enterprise documentation.
 
-## Information architecture
+## Structure of published content
+
+Using the same content structure across documentation provides consistent experience. Topics are listed from high level to more specific. For example, a new Grafana user may wish to learn conceptual information first, so Introduction is listed before installation.
+
+Nat all Topics are not used for every product. For example, Grafana OSS may use all of the headings, while Grafana Enterprise Traces only uses a subset.
+
+This table provides a list of the high-level topics used for grouping content. For example, a conceptual page explaining metrics would go under the Introduction topic.
+
+Italicized topics are optional and are usually found in specific contexts. For example, the Create, Manage, Monitor topics are used in Grafana OnCall but are not used in Grafana Tempo.
+
+| Topic | Example link | Contains |
+| --- | --- | --- |
+| Introduction | Introduction to Grafana | Conceptual information, fundamentals, architecture, etc. |
+| Get started | Get started with Tempo | Opinionated walk-throughs and tutorials |
+| Set up | Set up Loki | System requirements, Set up, configure, upgrade, migrate, etc. |
+| Configure |  | Configure may be it’s own directory if the number of pages |
+| Create alerts | Create alerts for Grafana OnCall | Specific to operational products such as Grafana OnCall. Not used with backend database products like Tempo and Loki. |
+| Manage alerts | Manage alerts for Grafana OnCall | Specific to operational products such as Grafana OnCall. Not used with backend database products like Tempo and Loki. |
+| Monitor alerts | Monitor alerts for Grafana OnCall | Specific to operational products such as Grafana OnCall. Not used with backend database products like Tempo and Loki. |
+| Integrate (with) product or Send data |  | How to set up data integrations, product integrations, data sources, clients, plugins, etc. |
+| Query data | TraceQL query editor | Query languages, query tools, examples |
```

Comment:
```suggestion
| _Query data_ | _TraceQL query editor_ | Query languages, query tools, and examples. |
```
This example might need a verb... "_Edit TraceQL queries_"

Reply:
And (HOW) to do that, one uses the TraceQL query editor, which can be described in the first paragraph of that topic.